app/uk/gov/hmrc/plasticpackagingtax/returns/forms/CommonFormValidators.scala

```diff
@@ -30,4 +30,21 @@ trait CommonFormValidators {
   val isEqualToOrBelow: (Option[String], Long) => Boolean = (value, limit) =>
     isNotEmpty(value) && isDigitsOnly(value) && value.exists(v => v.trim.toLong <= limit)
 
+  val isValidDecimal: String => Boolean =
+    (input: String) =>
+      try isNonEmpty(input) &&
+        BigDecimal(input.trim) >= 0 &&
+        BigDecimal(input.trim).scale <= 2
+      catch {
+        case _: java.lang.NumberFormatException => false
+      }
+
+  val isLowerThan: BigDecimal => String => Boolean = (threshold: BigDecimal) =>
+    (input: String) =>
+      try isNonEmpty(input) &&
+        BigDecimal(input.trim) < threshold
```

Comment:
I haven't tested that, just think we'd need to also check for negative numbers I think if you put -10.10 it'll still see it as valid value

Reply:
forget that now, seen that we validate it above, all good :)

Reply:
This has been addressed by using the isValidDecimal